src/main/java/org/opentripplanner/datastore/OtpDataStore.java

```diff
@@ -18,14 +18,7 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
-import static org.opentripplanner.datastore.FileType.CONFIG;
-import static org.opentripplanner.datastore.FileType.DEM;
-import static org.opentripplanner.datastore.FileType.GRAPH;
-import static org.opentripplanner.datastore.FileType.GTFS;
-import static org.opentripplanner.datastore.FileType.NETEX;
-import static org.opentripplanner.datastore.FileType.OSM;
-import static org.opentripplanner.datastore.FileType.REPORT;
-import static org.opentripplanner.datastore.FileType.UNKNOWN;
+import static org.opentripplanner.datastore.FileType.*;
```

Comment:
It seems the code doesn't follow the code style as described here https://github.com/opentripplanner/OpenTripPlanner/blob/master/docs/Codestyle.md

Reply:
Do you mean the entire code or just this line 21?

If you mean entire code we could check it according to the style ofc. Could you then reference few categories that caught your eye? I.e. the javadocs styles
Thanks.

Reply:
All of your changes. Seems like eg. the imports are in the wrong order. You can autoformat all of the changed code after importing the config to IDEA.

Reply:
What if the auto-formatting also modifies some of the original code dating back to 2019 and 2020 for example? Should we keep all of those changes or format only our own code additions?

Reply:
I think you can just highlight your changes, and reformat those parts, including the imports for the files changed.